### PR TITLE
fix: playbook parameter docs

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/docs/issues/224

not sure exactly what https://github.com/flanksource/docs/issues/224 was pointing to but I have made some improvements.

### CODE REVIEW

**Comment by moshloop** [comment]
Can we add snippet examples for each of the param types?

And maybe use a kitchen sink example with all options: 

https://github.com/flanksource/mission-control/commit/96c658fa68e8d74aabeadcfbc7f6d68c4c5b965c

File: mission-control/docs/reference/playbooks/parameters.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ title: Parameters
...
-| `required`    | Specify if the parameter is required                | `bool`                          |          |
-| `icon`        |                                                     | [`Icon`](/reference/types#icon) |          |
-| `description` | Description of the parameter.                       | `string`                        |          |
-| `type`        | Type of parameter. _(Defaults to "text")_           | `string`                        |          |
-| `properties`  | Properties of parameter. _Varies based on the type_ | `map[string]string`             |          |
+| Field         | Description                                         | Scheme                                         | Required |
+| ------------- | --------------------------------------------------- | ---------------------------------------------- | -------- |
+| `name`        | Name of parameter.                                  | `string`                                       | `true`   |
+| `default`     | Default value of the parameter.                     | [Go Template](/reference/scripting/gotemplate) |          |
```
Comment (moshloop) [inline mission-control/docs/reference/playbooks/parameters.md:14]: What is the context available ?

**Comment by moshloop** [comment]
@adityathebe - Can you also check any vale lint errors ? We probably need to force value to exit on error, and only run it on changed files with something like: https://github.com/marketplace/actions/changed-files

**Comment by adityathebe** [comment]
@moshloop added vale checks on changed files.

Added few more words on the ignore list and sorted it.

File: mission-control/docs/reference/playbooks/parameters.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+| Field         | Description                              | Schema                                             |
+| ------------- | ---------------------------------------- | -------------------------------------------------- |
+| `filter.type` | Limit the config items to the given type | [`ResourceSelector`](/reference/resource-selector) |
+
+### checks
+
+| Field         | Description                        | Schema                                             |
+| ------------- | ---------------------------------- | -------------------------------------------------- |
+| `filter.type` | Limit the checks to the given type | [`ResourceSelector`](/reference/resource-selector) |
+
```
Comment (moshloop) [inline mission-control/docs/reference/playbooks/parameters.mdx:113]: Are these correct ?

Comment (adityathebe) [reply mission-control/docs/reference/playbooks/parameters.mdx]: fixed it

**Pull request merged**